Documentation/learning/why.md

```diff
@@ -80,7 +80,7 @@ For distributed coordination, choosing etcd can help prevent operational headach
 ### Notes on the usage of lock and lease
 etcd provides [lock APIs][etcd-v3lock] which are based on [the lease mechanism][lease] and [its implementation in etcd][etcdlease]. The basic idea of the lease mechanism is: a server grants a token, which is called lease, to a requesting client. When the server grants a lease, it associates TTL with the lease. When the server detects passing of time longer than TTL, it revokes the lease. During the client holds a non revoked lease, it can claim that it owns an access to a resource associated with the lease. In the case of etcd, the resource is a key in the etcd keyspace. etcd provides lock APIs with this scheme. However, the lock APIs cannot be used as mutual exclusion mechanism. The APIs are called lock because of [the historical reason][chubby]. The lock APIs can be used an optimization mechanism of mutual exclusion as described below.
 
-The most important aspect of the lease mechanism is that TTL is defined as a physical time interval. Both of the server and client measures passing of time with their own clocks, which are not synchronized each other. It allows a situation that the server revokes the lease but the client still claims it owns the lease because of clock drift. Note that clock drift is not the only one reason which can cause such a situation. Other factors (e.g. long time pause caused by runtime GC, preemption by OS or VMM, etc) can introduce the problem. Actually, [jepsen][jepsen] can reproduce the situation even on a single machine.
+The most important aspect of the lease mechanism is that TTL is defined as a physical time interval. Both of the server and client measures passing of time with their own clocks. It allows a situation that the server revokes the lease but the client still claims it owns the lease.
```

Comment:
```suggestion
The most important aspect of the lease mechanism is that TTL is defined as a physical time interval. Both the server and client measure the passing of time with their own clocks. It allows the server to revoke a lease while the client still claims that it owns the lease.
```
The second sentence starting with "[i]t allows" does not read well. To what does "it" refer there? The independent measuring of elapsed time? The use of a "physical" time interval? The lease mechanism itself?